test/indexer/test_tdr.py

```diff
@@ -236,9 +244,21 @@ def _drs_file_id(self, file_id):
 class TestPlugin(tdr.Plugin):
     tinyquery: tinyquery.TinyQuery
 
+    def _set_utc_tzinfo(self, d: Optional[datetime]) -> Optional[datetime]:
```

Comment:
Nested function unless you think this could be generally useful for other tests.